src/app/kbv/controllers/self-assessment-payment.js

```diff
@@ -0,0 +1,36 @@
+const debug = require("debug")("load-question");
+const BaseController = require("hmpo-form-wizard").Controller;
+const { submitAnswer, getNextQuestion } = require("../service");
+
+class SelfAssessmentPaymentController extends BaseController {
```

Comment:
So it's clear this is a question controller and in line with the others, thoughts on renaming it to SelfAssessmentPayment**Question**Controller?